src/constants.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Literals for the PgBouncer machine charm."""
+
+PGB = "pgbouncer"
+PG = "postgres"
```

Comment:
Can this constant be used in the other files to avoid repeating its creation multiple times on those files?

Reply:
Good point, updated